docs/components/Featured.tsx

```diff
@@ -0,0 +1,67 @@
+import { Link } from "react-router-dom";
+
+const cards = [
+    {
+        title: "Getting Started",
+        description:
+            "Start your journey with Dojo and create your first application from the ground up.",
+        link: "/getting-started",
+    },
+    {
+        title: "Overview",
+        description:
+            "Discover the Dojo tools designed to streamline your application's build and deployment process.",
+        link: "/toolchain/katana",
+    },
+    {
+        title: "Architecture",
+        description:
+            "Explore the core architecture powering Dojo's onchain capabilities.",
+        link: "/architecture/execution-sharding",
```

Comment:
```suggestion
        link: "/framework",
```
Better to redirect to framework to match the description.